app/src/main/res/values/strings.xml

```diff
@@ -1,81 +1,179 @@
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_ua_mobile">Mobile</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_ua_vr">VR</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_reset">Reset Developer Settings</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_reset_button">Reset</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_environments">Environment</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_env_meadow">Meadow</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_env_cave">Cave</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_env_void">Void</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_pointer_color">Pointer Color</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_pointer_white">White</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_pointer_purple">Purple</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="on">ON</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="off">OFF</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="voice_search_start_1">What would you like to</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="voice_search_start_2">search on the web?</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="voice_search_example">Example: Please say, \"360 videos, weather, news…\"</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="voice_search_decoding">Searching…</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="voice_search_error">Sorry! I couldn\'t understand.</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="voice_search_try_again">Please try again.</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="voice_search_permission_1">Please allow microphone access to use the voice search.</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="search_placeholder">Search the Web or enter address</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="cancel_button">Cancel</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="ok_button">Ok</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="crash_dialog_header">We had an issue and crashed</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="crash_dialog_message">Please help Mozilla improve Firefox Reality by sending your crash data</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="crash_dialog_send_data">Always send data without asking</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="learn_more_button">Learn More</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="dont_sent_button">Don’t Send</string>
```

Comment:
we don't use a [curly quotation mark elsewhere](https://github.com/MozillaReality/FirefoxReality/pull/626/files#diff-01dafb3fd0217441330103cfc8300094R156) (personally, I'm a fan, but I'm unsure of the L10n implications)

Reply:
I removed all the contractions.